<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditOrdersDialog</class>
 <widget class="QDialog" name="AddEditOrdersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QAbstractItemView::item::selected {
	color: #eeeee4;
	background-color: #284256;
	border-color: #284256;
}

QDialog {
    height: 500px;
    width: 400px;
    background-color: #b59683;
  }
  
  #scrollArea #scrollAreaWidgetContents{
    background-color: #ebebd8;
  }

  #scrollArea{
    padding: 5px;
    border: 3px solid #ebebd8;
    border-radius: 15%;
    background-color: #ebebd8;
  }

  #tableWidget_services {
    min-height: 200px;
  }
  
  QLabel {
      color: #b59683;
      font-weight: bold;
  }
  
  QLabel[objectName*=&quot;title&quot;] {
    padding: 5px;
    
    font-size: 42px;
    font-weight: bold;
  
    color: #eeeee4;
    border-radius: 15%;
    background-color: #b59683;
    
    margin-bottom: 10px;
  }
  
  QPushButton {
      width: 90;
      height: 20;
      padding: 5px;
  
      font-weight: bold;
  
      color: #b59683;
      border: solid;
      border-width: 5px;
      border-radius:20px;
      border-color: #b59683;
      background-color: #eeeee4;
  }
  
  QPushButton::pressed {
        color: #eeeee4;
        background-color: #b59683;
  }
  
  QPushButton[objectName*=&quot;save&quot;], QPushButton[objectName*=&quot;add&quot;]
  {
    width: 90;
    height: 20;
    padding: 5px;
  
    font: bold;
  
    color: #eeeee4;
    border: solid;
    border-width: 5px;
    border-radius:20px;
    border-color: #284256;
    background-color: #284256;
  }
  
  QPushButton[objectName*=&quot;save&quot;]::pressed, QPushButton[objectName*=&quot;add&quot;]::pressed {
    color: #284256;
    background-color: #eeeee4;
  }
  
  QLineEdit {
      width: 200;
      height: 20;
      padding: 5px;
  
      font: bold;
  
      color: #284256;
      border: solid;
      border-width: 5px;
      border-radius:20px;
      border-color: #b59683;
      background-color: #eeeee4;
  }
  
  QLineEdit:focus {
      border-color: #284256;
  }
  
  QSpinBox, QDoubleSpinBox, QDateTimeEdit, QDateEdit {
      width: 150px; /* Добавляем единицу измерения (px) к значению ширины */
      height: 20px;
      padding: 5px;
  
      font: bold;
  
      color: #284256;
      border: 5px solid #b59683; /* Устанавливаем толщину границы */
      border-radius: 20px;
      background-color: #eeeee4;
  }
  
  QSpinBox:focus, QDoubleSpinBox:focus, QDateTimeEdit:focus, QDateEdit:focus {
    border-color: #284256;
  }
  
  QDoubleSpinBox::up-button, QDoubleSpinBox::down-button, QSpinBox::up-button, QSpinBox::down-button,
  QDateTimeEdit::up-button, QDateTimeEdit::down-button, QDateEdit::up-button, QDateEdit::down-button {
    width: 0;
    height: 0;
  }
  
  QPlainTextEdit {
      width: 200;
      height: 100;
      padding: 5px;
  
      font: bold;
  
      color: #284256;
      border: solid;
      border-width: 5px;
      border-radius:20px;
      border-color: #b59683;
      background-color: #eeeee4;
  }
  
  QPlainTextEdit:focus {
      border-color: #284256;
  }
  
  QComboBox {   
    width: 200;
    height: 20;
    padding: 5px;
  
    font: bold;
  
    color: #b59683;
    border: solid;
    border-width: 5px;
    border-radius:20px;
    border-color: #b59683;
    background-color: #eeeee4;
  }
  
  QComboBox:hover {
  
  }
  
  QComboBox::drop-down {
      subcontrol-origin: padding;
      subcontrol-position: top right;
      width: 15px;
      border-left: 5px solid #b59683;
  }
  
  QComboBox::down-arrow {
      
  }
  
  QComboBox QListView {
      padding: 5;
  }
  
  QComboBox::selected {
      color: #ebebd8;
      background-color: #284256;
  }
  
  QAbstractItemView {
  border:5px solid #b59683;
  border-radius: 15%;
  background-color: #eeeee4;
}

QAbstractItemView::item {
	color: #b59683;
	border: 1px solid #b59683;
}

QAbstractItemView::item::selected {
	color: #eeeee4;
	background-color: #284256;
	border-color: #284256;
}
  
  QAbstractItemView::selected {
      color: #eeeee4;
      background-color: #284256;
  }

QAbstractItemView QHeaderView {
	border:none;
}

QAbstractItemView QHeaderView::selected {
	
}


QAbstractItemView::section {
	padding: 5px;

	color:#eeeee4;
	background-color: #b59683;

	border: none;
	border-right: 1px solid #ebebd8;
}
  
  QGroupBox {
    color: #b59683;

    margin-top: 25px;

    border: solid;
    border-width: 3px;
    border-radius:20px;
    border-color: #b59683;
    background-color: #eeeee4;


  }

  QGroupBox::title {
    font-weight: bold;
  }
  </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-228</y>
        <width>381</width>
        <height>794</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_title">
         <property name="text">
          <string>Додавання</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QComboBox" name="comboBox_client">
             <property name="placeholderText">
              <string>Клієнт</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_managers">
             <property name="placeholderText">
              <string>Менеджер</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Дата та час отримання</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="dateTimeEdit_dateTimeOrder"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Дата виконання:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_dateCompletion">
           <property name="font">
            <font>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_services">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Послуга</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Кількість</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Вартість</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="pushButton_services_del">
           <property name="text">
            <string>Видалити</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_services_add">
           <property name="text">
            <string>Додати</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Загальна сумма:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_totalCost">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <double>999999.989999999990687</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QComboBox" name="comboBox_status">
           <property name="placeholderText">
            <string>Статус</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_comment">
         <property name="placeholderText">
          <string>Коментар</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>63</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_cancel">
           <property name="text">
            <string>Скасувати</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_save">
           <property name="text">
            <string>Зберегти</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
